core/src/banking_stage/vote_worker.rs

```diff
@@ -101,9 +114,6 @@ impl VoteWorker {
         banking_stage_stats: &mut BankingStageStats,
         slot_metrics_tracker: &mut LeaderSlotMetricsTracker,
     ) {
-        if self.storage.should_not_process() {
-            return;
-        }
```

Comment:
this makes received gossip votes process-able?

Reply:
gossip votes have always been processable because the vote storage was a shared object between tpu and gossip threads. Its' just that only the thread with the `tpu` receiver would actually do the processing.

Reply:
ah, that's right! Overlooked that they share storage.